pkg/csi/service/node.go

```diff
@@ -784,6 +741,114 @@ func (driver *vsphereCSIDriver) NodeGetInfo(
 	return nodeInfoResponse, nil
 }
 
+// initVolumeTopologyService is a helper method to initialize
+// TopologyService in node.
+func initVolumeTopologyService(ctx context.Context) error {
+	log := logger.GetLogger(ctx)
+	// This check prevents unnecessary RLocks on the volumeTopology instance.
```

Comment:
Im not sure this is needed.. why not just call `InitTopologyServiceInterface` directly? RLocks are inexpensive and can have multiple holders at the same time.